roles/irods/files/remote_psql_server_ca.crt

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----

```

Comment:
This looks like a file specific for a certain machine: that should not be located inside a `role` dir.
Shouldn't this be located here: `files/nibbler_cluster` ?